.env

```diff
@@ -0,0 +1,8 @@
+MYSQLDB_DATABASE=book_store
```

Comment:
git rm --cached .env - .env should never be in version control, add .env.template instead with actual values  removed - otherwise we would need to search for all  env var throughout the project to start it